.github/workflows/test-helm-book.yml

```diff
@@ -0,0 +1,238 @@
+name: Helm Docs Tutorials
+
+on:
+  push:
+    paths:
+      - "docs/book/src/cronjob-tutorial/testdata/project/**"
+      - "docs/book/src/getting-started/testdata/project/**"
+      - "docs/book/src/multiversion-tutorial/testdata/project/**"
+      - ".github/workflows/test-helm-book.yml"
+  pull_request:
+    paths:
+      - "docs/book/src/cronjob-tutorial/testdata/project/** "
+      - "docs/book/src/getting-started/testdata/project/**"
+      - "docs/book/src/multiversion-tutorial/testdata/project/**"
+      - ".github/workflows/test-helm-book.yml"
+
+jobs:
```

Comment:
@akagami-harsh, could we have a matrix since all tests are the same to simplify? 
Like: https://github.com/kubernetes-sigs/kubebuilder/blob/master/.github/workflows/test-e2e-book.yml

Reply:
Can you please also rebase it with master branch?